### ISSUE TITLE
Ratchet without ZeroMQ library

### ISSUE THREAD

**Issue by Shkarbatov** [ratchetphp/Ratchet#669]
Hi all!

Can I use Ratchet without ZeroMQ library?

**Comment by cboden** [comment]
> Can I use Ratchet without ZeroMQ library?

Yes, that tutorial is a demonstration, which was one of many ways to incorporate an evented system into a procedural system. 

> And what about ssl, can I work with it?

React's TCP library supports SSL, so yes. However, I would recommend using Nginx as a SSL terminator proxy. 

> Can i fork running php process with Ratchet and can children process restart if it will die?

I don't think you want to fork a process, but checkout [React's ChildProcess](https://github.com/reactphp/child-process).

**Comment by Shkarbatov** [comment]
> Yes, that tutorial is a demonstration, which was one of many ways to incorporate an evented system into a procedural system.

Can you please give me some link to example without ZeroMQ.
I need client connect from browser, and send data to client from back to browser.
like this: [https://github.com/Shkarbatov/WebSocketPHPWorkerman/blob/master/worker.php](https://github.com/Shkarbatov/WebSocketPHPWorkerman/blob/master/worker.php)

> I don't think you want to fork a process, but checkout React's ChildProcess.

Yes, correct, thanks!

**Comment by cboden** [comment]
Checkout [Pawl](https://github.com/ratchetphp/Pawl) while following the ZeroMQ tutorial. Just replace ZeroMQ with Pawl.

**Comment by Shkarbatov** [comment]
Is this correct?
https://github.com/Shkarbatov/WebSocketPHPRatchet

**Comment by cassifyit** [comment]
Hey! Thanks for reporting this issue 🙏

We're giving Ratchet some much-needed love and attention! As part of our issue cleanup initiative ([#1100](https://github.com/ratchetphp/Ratchet/issues/1100)), we're reviewing and closing long-standing issues to prioritize what our community currently needs most.

If this is still relevant to you, we'd love to continue this conversation in our [GitHub Discussions](https://github.com/ratchetphp/Ratchet/discussions). Just reference this issue when you post. If you want to support Ratchet, please consider [sponsoring our work](https://github.com/sponsors/reactphp)! ❤️
